daos/sqlite/Database.php

```diff
@@ -234,6 +234,12 @@ public function __construct() {
                         'INSERT INTO version (version) VALUES (11)'
                     ]);
                 }
+                if (strnatcmp($version, '12') < 0) {
+                    \F3::get('db')->exec([
+                        'CREATE UNIQUE INDEX uid ON new_items(uid, source);',
```

Comment:
The index should be created on `items`, there should be no table `new_items` at this point.

Reply:
7040883bfbd248dae6c5d266912379c31cfcacd5